spec/betterc.dd

```diff
@@ -0,0 +1,48 @@
+Ddoc
+
+$(SPEC_S Better C,
+
+`-betterC` is a Command-Line flag for `dmd`, which generated light-weight,
+barebone, runtime-free binaries with are suitable for linking from C.
```

Comment:
> The libc runtime is still linked, as far as I know.

I'm not sure if we emit any calls to C runtime functions directly from the compiler though. `memcpy` and `memset` are likely candidates.